lib/methods/import.js

```diff
@@ -6,13 +6,65 @@ const { EJSON } = require('bson');
 module.exports = self => {
   return {
     async import(req, moduleName) {
-      const {
-        docs, attachmentsInfo, exportPath, filePath
-      } = await self.readExportFile(req);
       if (!req.user) {
         throw self.apos.error('forbidden');
       }
 
+      const overrideLocale = self.apos.launder.boolean(req.body.overrideLocale);
+      let exportPath = self.apos.launder.string(req.body.exportPath);
+
+      let docs;
+      let attachmentsInfo;
+
+      if (exportPath) {
+        const filesData = await self.getFilesData(exportPath);
+
+        ({ docs, attachmentsInfo } = filesData);
```

Comment:
what means the fact that this method receives an exportPath? Does it mean that the gzip file has already been cleaned?